cli/src/alluxio.org/cli/cmd/exec/class.go

```diff
@@ -61,8 +61,10 @@ func (c *ClassCommand) Run(args []string) error {
 	} else {
 		c.JavaClassName = args[0]
 	}
-	javaArgs = append(javaArgs, args...)
-
+	if len(args) > 1 {
+		javaArgs = append(javaArgs, args[1:]...)
```

Comment:
if we don't handle the flags at all, it may be better to treat all the args as obscure args and put them in the `javaArgs`